src/vs/platform/terminal/common/terminal.ts

```diff
@@ -794,21 +795,6 @@ export interface ITerminalProfileSource extends IBaseUnresolvedTerminalProfile {
 	source: ProfileSource;
 }
 
-
-export interface ITerminalContributions {
-	profiles?: ITerminalProfileContribution[];
-	quickFixes?: ITerminalQuickFixContribution[];
-}
```

Comment:
Why did this move? Profiles certainly shouldn't be mentioned in a quick fix related file